doc/specs/#4191 - Formatted Copy/spec.md

```diff
@@ -0,0 +1,90 @@
+---
+author: Kayla Cinnamon @cinnamon-msft
+created on: 2020-04-01
+last updated: 2020-04-07
+issue id: #4191
+---
+
+# Formatted Copy
+
+## Abstract
+
+When copying text, the Terminal should provide the option of including formatting. Not all apps that receive text allow for picking which format you want when pasting. The default should be to only copy plain text, based on the response from this poll on Twitter.
+
+![Twitter poll](twitter-poll.png)
+
+## Solution Proposals
+
+A proposal for the right click behavior as well as two user settings proposals are described below:
+
+1. [Right click behavior](#right-click-behavior)
+2. [Settings option 1 - global setting](#settings-option-1---global-setting)
+3. [Settings option 2 - key binding argument](#settings-option-2---key-binding-argument)

```

Comment:
This has got me all confused as I'm leaving comments on the doc - maybe make "Right Click Behavior" option _3_, so that "Settings option 1" is _1_, etc